Cache_Redis.php

```diff
@@ -344,35 +344,42 @@ private function _get_accessor( $key ) {
 			$this->_accessors[$index] = null;
 		else {
 			try {
-				$server = $this->_servers[$index];
-				$accessor = new \Redis();
-
-				if ( substr( $server, 0, 5 ) == 'unix:' ) {
-					if ( $this->_persistent ) {
-						$accessor->pconnect( trim( substr( $server, 5 ) ),
-							null, null, $this->_instance_id . '_' . $this->_dbid );
+				if ( W3TC_REDIS_CLUSTER === true ) {
```

Comment:
A better check would be:
`if ( defined( 'W3TC_REDIS_CLUSTER' ) && W3TC_REDIS_CLUSTER ) {`